docs/README.md

```diff
@@ -125,52 +131,52 @@ Now, you are able to use all the blocks exported by the `disclosure-layout` app.
-| `initialVisibility` | `enum`                | `visible` to have it's content initially open, or `hidden` to be hidden.                                                                                                                        | `hidden`                                       |
-| `animated`          | `boolean` or `number` | To perform animations, you must set this to `true`. It'll enable additional data-\* attributes on it's content which you can use as selectors in CSS. It will also ensure that the element will | only get hidden when the transition has ended. | `false` |
+| `initialVisibility` | `enum`                | Defines the initial visibility of the layout content. Possible values are: `visible` (content initially open) or `hidden` (content is only displayed with user interaction).    | `hidden`   |
+| `animated`          | `boolean` | Whether or not the layout content should have animations. When set as `true`, this prop will enable additional `data-\*` attributes on the content which you can use as selectors in CSS. It will also ensure that the element will get hidden once the transition has ended. | `false` |
 
-### `disclosure-trigger` props
+#### `disclosure-trigger` props
 
 | Prop Name    | Type     | Description                                                                                          | Default value |
 | ------------ | -------- | ---------------------------------------------------------------------------------------------------- | ------------- |
-| `Show`       | `Block`  | This block will be rendered when prompt to show the content                                          | `undefined`   |
-| `Hide`       | `Block`  | This block will be rendered when prompt to hide the content                                          | `undefined`   |
-| `as`         | `string` | The HTML tag that will be used.                                                                      | `button`      |
-| `children`   | `Block`  | This block will be rendered if no `show` or `hide` is set                                            | `undefined`   |
-| `blockClass` | `string` | Block container class. This prop’s set value functions as a block identifier for CSS customizations. | `undefined`   |
+| `Show`       | `block`  | Name of the block that will be rendered when prompt to show the content. | `undefined`   |
+| `Hide`       | `block`  | Name of the block that will be rendred when prompt to hide the content.    | `undefined`   |
+| `as`         | `string` | HTML tag to be applied to the component when it is rendered on the UI. | `button`      |
+| `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `show` or `hide` props. | `undefined`   |
```

Comment:
```suggestion
| `children`   | `block`  | Name of the block that will be rendered in case no blocks are declared in the `Show` or `Hide` props. | `undefined`   |
```